docs/ref/models/constraints.txt

```diff
@@ -69,10 +69,31 @@ constraint.
 ``UniqueConstraint``
 ====================
 
-.. class:: UniqueConstraint(*, fields, name, condition=None, deferrable=None, include=None, opclasses=())
+.. class:: UniqueConstraint(*expressions, fields, name, condition=None, deferrable=None, include=None, opclasses=())
```

Comment:
Signature is outdated. We should also add `versionchanged` annotation and a [small release notes](https://docs.djangoproject.com/en/dev/releases/4.0/#miscellaneous) about signature changes.